hi-ml-histopathology/src/histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -38,19 +38,41 @@ def __init__(self,
                  root: Path,
                  dataset_csv: Optional[Union[str, Path]] = None,
                  dataset_df: Optional[pd.DataFrame] = None,
-                 occupancy_threshold: Optional[float] = None) -> None:
+                 occupancy_threshold: Optional[float] = None,
+                 random_selection: Optional[float] = None) -> None:
+        """
+        :param root: Root directory of the dataset.
+        :param dataset_csv: Full path to a dataset CSV file, containing at least
+        `TILE_ID_COLUMN`, `SLIDE_ID_COLUMN`, and `IMAGE_COLUMN`. If omitted, the CSV will be read
+        from `"{root}/{DEFAULT_CSV_FILENAME}"`.
+        :param dataset_df: A potentially pre-processed dataframe in the same format as would be read
+        from the dataset CSV file, e.g. after some filtering. If given, overrides `dataset_csv`.
+        :occupancy_threshold: A value between 0-1 such that only tiles with occupancy > occupancy_threshold
+        will be selected. If this is 0, all tiles are selected.
+        :random_selection: A value > 0 and <=1 such that this proportion of tiles will be randomly selected.
+        If this is 1, all tiles are selected.
```

Comment:
[Minor] The docs for `occupancy_threshold` and `random_selection` should mention what happens if `None` is given.

Reply:
Thanks, done